### DISCUSSION TITLE
ESP32 problem with channel 0 already used for display

### DISCUSSION THREAD

**Discussion by DWilhelmLKKS** [Arduino-IRremote/Arduino-IRremote#1223] [Help] [answered]
Hello community,

I'm using the following libraries for driving my [WT32-SC01 Plus](https://www.antratek.de/wt32-sco1-plus) (ESP32-S3) and it works really well:
- ArduinoJson by Benoit Blanchon v7.0.4
- LovyanGFX by lovyan03 v1.1.12
- lvgl by kisvegabor v9.1.0
- SD by Arduino, SparkFun v1.2.4

However, when I add IRremote (v4.3.1) for just sending signals the display always gets dark (after sending with `IrSender.sendNECRaw("0x6F905583")`).
Using IRremoteESP8266 v2.8.6 everything works fine but the library seems to need other codes and has less functionality.

This is my IR specific code:
```
// configuration
#define DISABLE_CODE_FOR_RECEIVER
#define ESP32
#define IR_SEND_PIN 10

// includes
#include <IRremote.h>

// initialize
void setup() {
  IrSender.begin(ENABLE_LED_FEEDBACK, USE_DEFAULT_FEEDBACK_LED_PIN);
}

// main
void loop() {
  lv_task_handler(); 
  lv_tick_inc(5);
  delay(5);
}
```

I think it could be a problem with a timer but the docs say that no timer is needed for sending signals...
Does somebody have a solution for this?

**Comment by ArminJo** [comment]
Ever tried
`disableLEDFeedback(); // Disable feedback LED at default feedback LED pin`

**Reply by DWilhelmLKKS** [reply]
I tried it now, changing my code to
```
// initialize
void setup() {
  IrSender.begin();
  disableLEDFeedback();
}
```
but it had no effect.

**Reply by ArminJo** [reply]
Then is pin 10 the wrong one!

**Reply by DWilhelmLKKS** [reply]
What do you mean with wrong pin?
Using another IR library with pin 10 (in combination with the display) works fine.
Even with IRremote the codes itself are working and I can control the remote device via my ESP.

**Reply by ArminJo** [reply, marked as answer]
Then try to change [line 1439 in IRTimer.hpp](https://github.com/Arduino-IRremote/Arduino-IRremote/blob/d8a851aba370c48e5dc51d48c9db730210e6d367/src/private/IRTimer.hpp#L1439)

**Reply by DWilhelmLKKS** [reply]
Thank you! You saved my life! 😄
That did the trick.
